### PR TITLE
Create containers.md

### PR DESCRIPTION
**Pull request by moolitayer**

### CODE REVIEW

**Comment by moolitayer** [comment]
@simon3z

**Comment by Fryguy** [comment]
(－‸ლ) I *always* mess up when I'm impersonating the bot

**Comment by Fryguy** [comment]
cc @blomquisg

File: providers/containers.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+## ManageIQ Container Providers
+
+### Containers Smart State Analysis
+
+Current architecture diagram:
+
+![container smart state analysis diagram](/images/containers-ssa.png "container smart state analysis")
+
+[Smart State Analysis demo with technical comments](Smart State Analysis demo with technical comments: http://file.rdu.redhat.com/~fsimonce/manageiq/containers-smart-state-analysis.webm)
```
Comment (simon3z) [inline providers/containers.md]: I'll nove this To YouTube and I'll vive you The link.

**Comment by moolitayer** [comment]
Added a link from architecture/providers.md

**Comment by simon3z** [comment]
@moolitayer https://www.youtube.com/watch?v=Uc5kTSdVNZA

**Comment by moolitayer** [comment]
Use youtube video

**Comment by simon3z** [comment]
Is this ready for merge? cc @Fryguy

**Pull request merged**